lua/goto-preview.lua

```diff
@@ -120,6 +135,12 @@ M.apply_default_mappings = function()
       "<cmd>lua require('goto-preview').goto_preview_definition()<CR>",
       { noremap = true }
     )
+    vim.api.nvim_set_keymap(
+      "n",
+      "gpt",
+      "<cmd>lua require('goto-preview').goto_preview_type_implementation()<CR>",
```

Comment:
You probably meant to have this instead?
```suggestion
      "<cmd>lua require('goto-preview').goto_preview_type_definition()<CR>",
```

Reply:
Ahh yes, I'm sorry, I wasn't aware of that